modules/oxide/src/less/theme/content/footnotes/footnotes.less

```diff
@@ -0,0 +1,32 @@
+//
+// Footnotes
+//
+
+sup.mce-footnote a {
+  color: black;
+  text-decoration: none;
+}
+
+div.mce-footnotes {
+  hr {
+    margin-left: 0;
+    margin-right: auto;
```

Comment:
Should this be a `margin-inline-start` and `margin-inline-end` to support rtl content as well?

Reply:
Figured it out now, thanks for the tip!